packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -300,6 +310,17 @@ export default {
 
         shouldShowErrorPage () {
             return !this.hasCheckoutLoadedSuccessfully;
+        },
+
+        shouldShowErrorDialog () {
+            if (this.nonFulfillableError) {
```

Comment:
Should be able to do the below to make it a one liner.

`this.nonFulfillableError?.shouldShowInDialog`

or

`this.nonFulfillableError && this.nonFulfillableError.shouldShowInDialog`